Sources/SwiftSyntax/CustomTraits.swift

```diff
@@ -0,0 +1,80 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+extension ActorDeclSyntax {
+  var introducer: TokenSyntax {
+    get {
+      return actorKeyword
+    }
+    set {
+      actorKeyword = newValue
+    }
+  }
+}
+
+extension ClassDeclSyntax {
+  var introducer: TokenSyntax {
+    get {
+      return classKeyword
+    }
+    set {
+      classKeyword = newValue
+    }
+  }
+}
+
+extension EnumDeclSyntax {
+  var introducer: TokenSyntax {
+    get {
+      return enumKeyword
+    }
+    set {
+      enumKeyword = newValue
+    }
+  }
+}
+
+extension ExtensionDeclSyntax {
+  var introducer: TokenSyntax {
+    get {
+      return extensionKeyword
+    }
+    set {
+      extensionKeyword = newValue
+    }
+  }
+}
+extension ProtocolDeclSyntax {
```

Comment:
Very nitpicky: Above you have a newline between the extensions, here you don’t.

Reply:
Ah thanks, copy paste strikes again :)